index.bs

```diff
@@ -3384,6 +3384,8 @@ To <dfn>locate nodes using inner text</dfn> with given |context nodes|,
 
 1. For each |context node| in |context nodes|:
 
+  1. If |context node| does not implement {{HTMLElement}}, [=break=].
```

Comment:
The innerText algorithm seem to recurse only if there is a match? this is probably not sufficient since I imagine we want to recurse for Document and DocumentFragment nodes at least

Reply:
I think you want to continue, not break? And yes, I think you want a specific step for `Document` and `DocumentFragment`. So overall something like:
```
1. If |context node| implements {{Document}} or {{DocumentFragment}}:
 
  1. Let |child nodes| be an empty [=/list=]
  
  1. For each node |child| in the [=children=] of |context node|
  
    1. [=list/Append=] |child| to |children|.

  1. [=Extend=] |returned nodes| with the result of [=trying=] to [=locate nodes using inner text=] with |child nodes|, |selector|, |max depth|, |match type|, |ignore case|, and |maximum returned node count|.
  
1. If |context node| does not implement {{HTMLElement}} then [=continue=].
```

Then further down you could eliminate the Element checks before calling the algorithm recursively.

I also note that in the recursive calls we need to use `[=trying=]` because it returns a result type (even though recursive calls are actually infallible).